### PR TITLE
remove translate code from projects without translation

### LINKED ISSUE #2877: User Management does not load on project without translation
* **Overview of the issue**
User Management does not load on project without translation enabled.  No stack trace is thrown in the console but the issue is in user-management.js in the resolve methods.

* **Motivation for or Use Case** 
I use JHipster without translation

* **JHipster Version(s)** 
Master - this is not an issue in the current 2.27.0 release.

* **JHipster configuration, a `.yo-rc.json` file generated in the root folder** 
Any project with translation set to no

* **Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory** 
No entities required

* **Browsers and Operating System** 
Issue with all browsers and operating systems

* **Reproduce the error** 
Generate a project without translation.  'translatePartialLoader' is included by default in user-management.js routes of user-management and user-management-detail

* **Suggest a Fix** 
#2878

### PR DESCRIPTION
**Pull request by ruddell**
Fix #2877

### CODE REVIEW

File: generators/client/templates/src/main/webapp/app/admin/user-management/_user-management.js [hunk 1]
```diff
@@ -17,10 +17,12 @@ angular.module('<%=angularAppName%>')
                     }
                 },
                 resolve: {
```
Comment (andidev) [inline generators/client/templates/src/main/webapp/app/admin/user-management/_user-management.js]: Could you place the if-cases arround the resolve property instead i.e.
```javascript
                views: {
                    'content@': {
                        templateUrl: 'app/admin/user-management/user-management.html',
                        controller: 'UserManagementController'
                    }
                }<% if (enableTranslation) { %>,
                resolve: {
                    translatePartialLoader: ['$translate', '$translatePartialLoader', function ($translate, $translatePartialLoader) {
                        $translatePartialLoader.addPart('user-management');
                        return $translate.refresh();
                    }]
                }<% } %>
            })
```

File: generators/client/templates/src/main/webapp/app/admin/user-management/_user-management.js [hunk 2]
```diff
@@ -37,10 +39,12 @@ angular.module('<%=angularAppName%>')
...
+                <% if (enableTranslation) { %>
                     translatePartialLoader: ['$translate', '$translatePartialLoader', function ($translate, $translatePartialLoader) {
                         $translatePartialLoader.addPart('user-management');
                         return $translate.refresh();
                     }]
+                <% } %>
```
Comment (andidev) [inline generators/client/templates/src/main/webapp/app/admin/user-management/_user-management.js]: same here

**Comment by ruddell** [comment]
Thank you for the feedback, changes made

**Comment by andidev** [comment]
Thanks merging this right away!

**Comment by deepu105** [comment]
This need not be done this way, anyway let me check

**Pull request merged**